### PR TITLE
Task - 11, 13 and 36

### LINKED ISSUE #11: Task 8 (icon box)
Design a card similar to the "Bordered Featured Boxes" found in the page "featured-boxes.html" of the Canvas theme. 

- The main class for this design is called *s-ibox-col* and define in in a new file at `/styles/_iboxes.scss`
- Define an html file to display it at /tasks/task-008/ibox.html. The html structure should follow this format:

```html
<article class="s-ibox-col">
  <section class="s-ibox-col__icon"><i class="i-some-icon"></i></section>
  <section class="s-ibox-col__content">
    <h1> This is title, big black </h1>
    <p> This is description, grey color </p>
  </section>
</article>
```

### PR DESCRIPTION
**Pull request by navjot-chahal**
@pmvald I have committed all the changes to the branch and have to file a single pull request for all. The stash option will not work for me as the changes are already committed to my branch.

I understand this is not the protocol that should be followed. I apologize for this mess but I will make sure to complete one task at a time followed by a single pull request in the future.

This will close #11, #10, and #54.

### CODE REVIEW

File: tasks/task-036/news.html [hunk 1]
```diff
@@ -0,0 +1,1055 @@
+<!DOCTYPE HTML
```
Comment (pmvald) [inline tasks/task-036/news.html:1]: The only file I need is this file. Other changes you made are related to tasks that were not supposed to do by interns. To make things simple I'm gonna make a new branch for you, and you just include this file into your branch. Now as for this file, it is tooo large! It should be much much smaller and more organized. I see lot of inline style in it (see below codes). How did you generate this?

File: tasks/task-036/news.html [hunk 2]
```diff
@@ -0,0 +1,1055 @@
...
+
+<head>
+  <!--[if gte mso 9]>
+<xml>
+  <o:OfficeDocumentSettings>
+    <o:AllowPNG/>
+    <o:PixelsPerInch>96</o:PixelsPerInch>
+  </o:OfficeDocumentSettings>
+</xml>
+<![endif]-->
```
Comment (pmvald) [inline tasks/task-036/news.html:14]: ??

File: tasks/task-036/news.html [hunk 3]
```diff
@@ -0,0 +1,1055 @@
...
+  <!--[if !mso]><!-->
+  <link href="https://fonts.googleapis.com/css?family=Cabin:400,700&display=swap" rel="stylesheet" type="text/css">
+  <link href="https://fonts.googleapis.com/css?family=Lato:400,700&display=swap" rel="stylesheet" type="text/css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap" rel="stylesheet" type="text/css">
+  <!--<![endif]-->
+
+</head>
+
+<body class="clean-body"
+  style="margin: 0;padding: 0;-webkit-text-size-adjust: 100%;background-color: #e7e7e7;color: #000000">
```
Comment (pmvald) [inline tasks/task-036/news.html:226]: no inline style

**Review by pmvald — COMMENTED** [review]
@navjot-chahal I made code review.

**Comment by pmvald** [comment]
@navjot-chahal Please use branch navjot2 to make a PR for the html email. Leave other tasks out.

File: tasks/task-036/news.html (hunk 1 above)
Comment (navjot-chahal) [reply tasks/task-036/news.html:1]: I think the inline style was present in the template I downloaded.

**Pull request closed without merging**